test/unit/exception_handling/mailer_test.rb

```diff
@@ -3,7 +3,11 @@
 module ExceptionHandling
   class MailerTest < ActionMailer::TestCase
 
-    include ActionDispatch::Assertions::SelectorAssertions
+    if ::Rails.const_defined?(:Dom)
```

Comment:
That's a good point. Since this branch will only run for Rails 4 we won't need to check and can just keep the `include ::Rails::Dom::Testing::Assertions::SelectorAssertions` statement by itself. I'll go ahead and fix that